Hogajama/hogajama-rs/src/main/java/com/gepardec/hogarama/rest/unitmanagement/UnitApiImpl.java

```diff
@@ -39,6 +39,41 @@ public Response create(SecurityContext securityContext, UnitDto unitDto) {
         Unit unit = translator.fromDto(unitDto);
         service.createUnit(unit);
 
-        return new BaseResponse<>(HttpStatus.SC_CREATED).createRestResponse();
+        return new BaseResponse<>(unitDto, HttpStatus.SC_CREATED).createRestResponse();
+    }
+
+    @Override
+    @Transactional
+    public Response update(String id, SecurityContext securityContext, UnitDto unitDto) {
+        LOG.info("Updating unit with id {}.", id);
```

Comment:
Wenn schon ein Log geschrieben wird, kann man auch gleich ein paar Infos mehr ausgeben, z.B. unitDto.toString()

Reply:
Update in 466e3ca17597fc231fda06cfad2fa65238206510